### ISSUE TITLE
documentation for data packs

### ISSUE THREAD

**Issue by GretaTimaite** [udsleeds/openinfra#130]
This is an idea that crossed my mind after reading @hulsiejames's outline for the vignette #116 

> Though, this also required the data_packs vignette to be updated... my current thoughts are:
>    re-write the data_packs vignette to have the following format:
    a brief intro on motivation and context for transport infrastructure data packs,
    followed by - visualisations of a 1/2.5km circular buffer around the city of leeds showing data pack outputs. Each visualisation has descriptions of data pack column names, what they mean, and potentially how they are derived... leading onto the next point:
    followed by reproducible code on how to create the plots (or at least a link to another vignette that goes through this in more detail) so people/planners can visualise their own data packs (obtainable from the project releases page)

This is a lot of information for a single vignette, so I thought maybe it would make sense to use `bookdown` package and write an report/article divided in chapters and sub-chapters?

I think it would make the information much more accessible and I also assume it might help to avoid some of the issues, such as taking too long to load a vignette because of many embedded interactive maps? 

I do realise it would be more time consuming, so obvs it would be better to have an imperfect vignette for now, but maybe it's something to consider.

**Comment by hulsiejames** [comment]
Yes I like the sound of this! To be honest, with time constraints I will first get the vignette fixed up now and this is added to the TODO!

**Comment by hulsiejames** [comment]
First step towards this (in comming days...)

Remove the [**openinfra function definitions**](https://udsleeds.github.io/openinfra/articles/data_packs.html#data-packs-openinfra-function-definitions) from the transport data packs vignette and set this up as its own vignette.  

Then:
A - Update this section to include definitions of alll functions (currently only 3/4)
B - For each function, add justifications (tags used) and descriptions of functions, can also link to [function docs](https://udsleeds.github.io/openinfra/reference/index.html)

**Comment by hulsiejames** [comment]
This has now been split in two. 

[data_packs.Rmd](https://github.com/udsleeds/openinfra/blob/main/vignettes/data_packs.Rmd) Covers the introduction to data packs (_needs re-writing_), example visualisations and data pack column name descriptions.

[data_packs_documentation.Rmd](https://github.com/udsleeds/openinfra/blob/main/vignettes/data_packs_documentation.Rmd) Now contains documentation informatio, including the general workflow used to create data packs, an exmaple production for Leeds, documentation for the openinfra functions used to create a data pack.

If there are no other suggestions, I will close this issue after I have re-written a better introduction (motivation etc.) for [data_packs.Rmd](https://github.com/udsleeds/openinfra/blob/main/vignettes/data_packs.Rmd).
